pubtools/pulplib/_impl/model/task.py

```diff
@@ -66,11 +67,11 @@ class Task(PulpObject):
     """The ID of the repository associated with this task, otherwise None."""
 
     units = pulp_attrib(
-        default=attr.Factory(FrozenList),
-        type=FrozenList,
+        default=attr.Factory(frozenlist),
+        type=list,
```

Comment:
This has been refactored from frozenlist to a builtin list. Looks like the change is intentional, but I'm pointing it out just in case it isn't.